lib/Mojo/Server/Morbo/Backend.pm

```diff
@@ -0,0 +1,60 @@
+package Mojo::Server::Morbo::Backend;
```

Comment:
I think these classes could be a generic component. How about naming them "Mojo::File::Watch" or something?